sketch/edge_length.py

```diff
@@ -0,0 +1,286 @@
+# ███╗   ██╗██████╗
+# ████╗  ██║██╔══██╗
+# ██╔██╗ ██║██║  ██║
+# ██║╚██╗██║██║  ██║
+# ██║ ╚████║██████╔╝
+# ╚═╝  ╚═══╝╚═════╝
+#
+# ND (Non-Destructive) Blender Add-on
+# Copyright (C) 2024 Tristan S. & Ian J. (HugeMenace)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# ---
+# Contributors: Shaddow
+# ---
+
+import bpy
+import bmesh
+from math import radians
+from .. lib.base_operator import BaseOperator
+from .. lib.overlay import init_overlay, register_draw_handler, unregister_draw_handler, draw_header, draw_property, draw_hint
+from .. lib.events import capture_modifier_keys, pressed
+from .. lib.preferences import get_preferences, get_scene_unit_factor
+from .. lib.numeric_input import update_stream, no_stream, get_stream_value, new_stream, has_stream, set_stream
+from .. lib.objects import get_real_active_object
+from .. lib.polling import ctx_edit_mode, obj_is_mesh, ctx_objects_selected, app_minor_version
+from .. lib.math import v3_distance, v3_average
+from .. lib.points import init_points, register_points_handler, unregister_points_handler
+
+
+
+class ND_OT_edge_length(BaseOperator):
+    bl_idname = "nd.edge_length"
+    bl_label = "Set edge length"
+    bl_description = """Allows you to accuratly set the distance between two selected vertices or n not connected selected edges"""
```

Comment:
Simplify description:

```suggestion
    bl_description = """Accurately define or offset the length of any selected disconnected edges"""
```